aws-lc-rs/src/signature.rs

```diff
@@ -313,6 +313,7 @@ pub trait VerificationAlgorithm: Debug + Sync + sealed::Sealed {
     //
     /// # Errors
     /// `error::Unspecified` if inputs not verified.
+    #[cfg_attr(docsrs, doc(cfg(feature = "ring-sig-verify")))]
```

Comment:
you could consider `#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]` at the top level of the crate, which removes the needs to write these manually -- it automatically extracts the same information from the `cfg(feature = )` gates themselves.

Reply:
Ah I didn't realize it could figure that out automatically if you put it at the crate level. Thanks!